tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -58,6 +58,7 @@ std::unique_ptr<TestSetup> TestSetup::ParseArgs(int argc, char** argv) {
     replicaConfig.clientBatchingEnabled = true;
     replicaConfig.pruningEnabled_ = true;
     replicaConfig.numBlocksToKeep_ = 10;
+    replicaConfig.timeServiceEnabled = true;
```

Comment:
Is there any reason on earth we would like to test when timeServiceEnabled = false? Or current code is obsolete?

Reply:
At the moment I do not see any reason for that. Once we need it, we'll add another parameter to the tester replica.